pom.xml

```diff
@@ -223,6 +223,17 @@
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>com.tngtech.archunit</groupId>
+      <artifactId>archunit-junit5</artifactId>
+      <version>0.13.1</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.github.java-diff-utils</groupId>
+      <artifactId>java-diff-utils</artifactId>
+      <version>4.5</version>
+    </dependency>
```

Comment:
Shouldn't this scope be test too?

Reply:
Yes, sorry I missed it. I have added it.